src/NServiceBus.Gateway/Config/SiteConfig.cs

```diff
@@ -72,6 +72,22 @@ public bool LegacyMode
                 this["LegacyMode"] = value;
             }
         }
+
+        /// <summary>
+        /// Is the site behind a reverse proxy 
+        /// </summary>
+        [ConfigurationProperty("UsesReverseProxy", IsRequired = false, DefaultValue = false, IsKey = false)]
+        public bool UsesReverseProxy
```

Comment:
As we're moving towards a config model with code api's only I would prefer to not expose this here? (and just go with the code first api)

Reply:
Yeah, I'm happy to remove that, we're using code api's to configure anyway. Will give it a bit to see if any more feedback then update.